lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -943,6 +943,28 @@ class DecomposeAtenNativeBatchNormOp
 };
 } // namespace
 
+// Decompse `Aten_UnsafeViewOp` into `AtenViewOp`. _unsafe_view() differs from
```

Comment:
Please add the reference here: https://github.com/pytorch/pytorch/blob/364055b2771ecf9b54f1d67a8bf44bb5496476d4/aten/src/ATen/native/TensorShape.cpp#L2072